.github/scripts/validate-deploy.sh

```diff
@@ -47,6 +47,7 @@ check_k8s_namespace "${NAMESPACE}"
 
 check_k8s_resource "${NAMESPACE}" "sealedsecret" "sonarqube-access"
 check_k8s_resource "${NAMESPACE}" "secret" "sonarqube-access"
-
+sleep 7m
+kubectl rollout status "statefulset/"${STATEFULSET} -n "${NAMESPACE}" || exit 1
```

Comment:
This can be replaced with a call to `check_k8s_resource "${NAMESPACE}" statefulset "${STATEFULSET}"`